js/todoTrigger.js

```diff
@@ -0,0 +1,79 @@
+import { addTodo } from './addTodo.js';
+import { checkTodo } from './checkTodo.js';
+import { deleteTodo } from './deleteTodo.js';
+import { editTodo } from './editTodo.js';
+
+const $todoInput = document.querySelector('#new-todo-title');
+const $todoList = document.querySelector('.todo-list');
+
+const $todoApp = document.querySelector(".todoapp");
+const $todoCount = $todoApp.querySelector(".todo-count > strong");
+
+const $viewAllList = $todoApp.querySelector('.all');
+const $viewTodoList = $todoApp.querySelector('.active');
+const $viewDoneList = $todoApp.querySelector('ul.filters').querySelector('.completed');
```

Comment:
```suggestion
const $viewDoneList = $todoApp.querySelector('ul.filters .completed');
```
querySelector 내부는 css selector syntax를 따르는점 참고하시구요!